gearcmd/worker.go

```diff
@@ -114,6 +114,9 @@ func (conf TaskConfig) doProcess(job baseworker.Job) error {
 		// Will be nil if the channel was closed without any errors
 		return err
 	case <-time.After(conf.CmdTimeout):
+		if err := cmd.Process.Kill(); err != nil {
```

Comment:
In this case, we're explicitly sending SIGKILL to the process because it's been hanging too long and something is probably wrong.